docs/source/noise/index.rst

```diff
@@ -186,7 +186,7 @@ Noise types for each column
     - Borrow a social security number, leave a field blank, write the wrong digits, make typos, make OCR errors
     - Note that 'borrow a social security number' only applies to the W-2 and 1099 dataset.
       In the 1040 form, the same noise types apply to the SSN columns for the joint filer and dependents
-  * - Income 
+  * - Wages 
```

Comment:
I'd assume we'd need both in this table, since we have income in the 1040?

Reply:
I thought we removed any mention of income from 1040?

Reply:
Ah, you're right; I forgot that we cut that for this release.